config/config.go

```diff
@@ -140,8 +140,28 @@ func NewDefaultConfig() *AgentConfig {
 				Default: "redis",
 			},
 		},
+		CMAB: CMABConfig{
+			Enabled:            false,
+			PredictionEndpoint: "https://prediction.cmab.optimizely.com",
+			RequestTimeout:     10 * time.Second,
+			Cache: map[string]interface{}{
+				"type": "memory",
+				"size": 1000,
+				"ttl":  "30m",
+				"redis": map[string]interface{}{
+					"host":     "localhost:6379",
+					"password": "",
+					"database": 0,
+				},
```

Comment:
default type is "memory". We may not need "redis" config here.

Reply:
Good point! 
I was pre-ampting if we wanted to include redis later, like with UPS. But went too far ahead of myself.
We don't need redis.